class/keycloak.yml

```diff
@@ -9,3 +14,57 @@ parameters:
           - keycloak/component/main.jsonnet
         input_type: jsonnet
         output_path: keycloak/
+      - output_path: keycloak/01_keycloak_helmchart
+        input_type: helm
+        output_type: yaml
+        input_paths:
+          - keycloak/helmcharts/keycloak
+        helm_values:
+          statefulsetLabels: ${keycloak:labels}
+          resources: ${keycloak:resources}
+          # extraEnv *MUST* be a string, as it's fed through a templating
+          # function.
+          extraEnv: |
+            - name: JAVA_OPTS
+              value: >-
+                        -XX:+UseContainerSupport
+                        -XX:MaxRAMPercentage=50.0
+                        -Djava.net.preferIPv4Stack=true
+                        -Djboss.modules.system.pkgs=$JBOSS_MODULES_SYSTEM_PKGS
+                        -Djava.awt.headless=true
+                        ${keycloak:extraJavaOpts}
+            - name: KEYCLOAK_STATISTICS
+              value: ${keycloak:monitoring:statistics}
+          extraEnvFrom: |
+            - secretRef:
+                name: ${keycloak:admin:secretname}
+            - secretRef:
+                name: $[ self:keycloak:postgres:external:secretname if exports:keycloak:postgres:builtin == false ]
+          serviceAccount:
+            labels: ${keycloak:labels}
+          ingress:
+            enabled: ${keycloak:ingress:enabled}
+            labels: ${keycloak:labels}
+            rules:
+              - host: ${keycloak:hostname}
+            tls:
+              - host: ${keycloak:hostname}
+          route:
+            enabled: ${keycloak:route:enabled}
+            labels: ${keycloak:labels}
+            host: ${keycloak:hostname}
+          service:
+            labels: ${keycloak:labels}
+          serviceMonitor:
+            enabled: ${keycloak:monitoring:enabled}
+            labels: ${keycloak:labels}
+          prometheusRule:
```

Comment:
I don't really see the use of this. By default there are no rules defined. At least a parameter should be exposed to setup rules.
https://github.com/codecentric/helm-charts/blob/master/charts/keycloak/values.yaml#L336

Reply:
I was under the (mistaken) assumption that the Chart would provide some useful default rules when settting `prometheusRule.enabled` to `true`.